packages/wrapper-react/src/index.tsx

```diff
@@ -49,7 +49,7 @@ export const MonacoEditorReactComp: React.FC<MonacoEditorProps> = (props) => {
         const startMonaco = async () => {
             if (containerRef.current) {
                 try {
-                    wrapperRef.current.registerTextChangeCallback(onTextChanged);
+                    wrapperRef.current.registerTextChangedCallback(onTextChanged);
```

Comment:
The new dependency array will always preserve the old reference of onTextChanged, can we create a callback and wrap the onTextChanged
i.e.
```tsx
const onTextChangedRef = useRef(onTextChanged);
onTextChangedRef.current = onTextChanged;
```
and here
```tsx
wrapperRef.current.registerTextChangedCallback(() => onTextChangedRef.current());
```
this way we register only once but always call the new callback (users also don't need to put onTextChanged in useCallback)

Reply:
Yes, thank you, that sounds like a good idea.

Reply:
Done,. see ⬇️